gnd/src/main/java/com/google/android/gnd/persistence/local/room/LocalDatabase.java

```diff
@@ -36,13 +36,14 @@
       RecordMutationEntity.class
     },
     // TODO(#128): Reset version to 1 before releasing.
```

Comment:
Is this TODO still valid?  
Can we extract this value to a properties file instead or is any strong reason why needs to be in the code?  I would forget to reset it if I was making a release :\

Reply:
Unfortunately values in Java annotations must be constants. The version increment is a peculiarity of Room; we increment it in development so that the old version of the local db will get wiped each time. Once we release, we should reset it to 1 and then provide migration scripts between versions each time we change the local schema.